addons/account/wizard/setup_wizards.py

```diff
@@ -45,8 +53,16 @@ def write(self, vals):
         # fields, the inverse write is done one value at a time, and thus the constraint is verified
         # one value at a time... so it is likely to fail.
         for wiz in self:
+            if wiz.opening_date:
+                year = wiz.opening_date.year
+            else:
+                year = datetime.now().year
+            if wiz.company_id.fiscalyear_last_day == 29 and wiz.company_id.fiscalyear_last_month == '2':
```

Comment:
Why does this seem like duplicated code when my point was to remove duplicated code?

Reply:
Hello @william-andre 

Changes have been done!

Can you please check?